src/totals.ts

```diff
@@ -145,8 +145,12 @@ export const getTotals = (
   // Ritenuta ENASARCO
   if (gestione_contributo_previdenziale && contributo_previdenziale.tipo === 'TC07' && contributo_previdenziale.enasarco?.enabled) {
     it.ritenuta_enasarco = (temp_subtotal * parseFloat(contributo_previdenziale.enasarco.percentuale)) / 100;
+    // Riaggiunge al totale il contributo previdenziale
+    total += it.contributo_previdenziale;
```

Comment:
@lucalategola Qui lo aggiungi perché alla riga 131 viene tolto?

```
  // ENASARCO -> togliere importo dal totale anziché sommare
  if (gestione_contributo_previdenziale && contributo_previdenziale.tipo === 'TC07') {
    total -= it.contributo_previdenziale;
  } else {
    total += it.contributo_previdenziale;
  }
```

Reply:
Ho capito, servono entrambi perché `it.contributo_previdenziale` va azzerato solo se `contributo_previdenziale.enasarco?.enabled = true`. A posto così.